layers-samples/proj-multiview-occlusion.html

```diff
@@ -162,10 +162,21 @@
         + event.session.visibilityState);
     }
 
+    function onSelectStart(ev) {
+      ev.frame.session.pauseDepthSensing()
```

Comment:
What I did in my local Chromium test pages was added a button (though using select is a good idea too), to trigger a toggle based on depthActive. This may be a little more ergonomic for testing as you don't have to hold the "select" down for it to be paused. Not blocking, but just a thought to consider!

Reply:
Is there an easy way to draw a button with the webxr samples renderer?

Reply:
Oh, I was suggesting that you do something like:
```
let session = ev.frame.session;
session.depthActive ? session.pauseDepthSensing() : session.resumeDepthSensing();
```

In e.g. SelectStart and remove SelectEnd altogether.